project.html

```diff
@@ -4,38 +4,38 @@
 permalink: project
 ---
 
-<p><strong>KeePassXC</strong> is a modern, secure, and open-source password manager that stores and manages your most sensitive information.</p>
+<p><strong>KeePassXC</strong> is a modern, secure, and open-source password manager that stores and manages your most sensitive info.</p>
 
-<p>You can run KeePassXC on Windows, macOS, and Linux systems. KeePassXC is for people with extremely high demands of secure
-personal data management. It saves many different types of information, such as usernames, passwords, URLs, attachments,
-and notes in an offline, encrypted file that can be stored in any location, including private and public cloud solutions.</p>
+<p>For Linux, macOS and Windows. Save many different types of info (such as usernames, passwords, URLs, attachments,
```

Comment:
Again, I really don't like info. Why not information or data?

Reply:
Data is less tangible. I think that is a judgement call.